WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt

```diff
@@ -0,0 +1,27 @@
+this_is_not_sparta = { # Spartan cultured character is a coward, AKA unthinkable in Spartan culture
+	icon = 90
+
+	general_opinion = -30
+	monthly_character_prestige = -1
+
+	martial = -20 # Prevents a Spartan AI ruler from choosing characters with the Craven Trait as Commanders
```

Comment:
Could we go into the Commander Minor Title and prevent people with this modifier from getting the office?